src/main/webapp/app/exercises/programming/manage/grading/feedback-analysis/Modal/feedback-detail-channel-modal.component.html

```diff
@@ -0,0 +1,87 @@
+<div class="modal-header">
+    <h4 class="modal-title">
+        <span [jhiTranslate]="TRANSLATION_BASE + '.title'"></span>
+    </h4>
+    <button type="button" class="btn-close" aria-label="Close" (click)="dismissModal()"></button>
+</div>
+<div class="modal-body">
+    <div class="mb-3 text-muted">
+        <p [jhiTranslate]="TRANSLATION_BASE + '.description'"></p>
+    </div>
+    <form [formGroup]="form" (ngSubmit)="submitForm(false)">
+        <div class="form-group">
+            <label for="name" [jhiTranslate]="TRANSLATION_BASE + '.label'"></label>
+            <input type="text" class="form-control" id="name" formControlName="name" [attr.placeholder]="TRANSLATION_BASE + '.placeholder' | artemisTranslate" />
+            @if (form.get('name')?.invalid && (form.get('name')?.dirty || form.get('name')?.touched)) {
+                <div class="alert alert-danger mt-2">
+                    @if (form.get('name')?.errors?.['required']) {
+                        <div [jhiTranslate]="TRANSLATION_BASE + '.requiredValidationErrorName'"></div>
+                    }
+                    @if (form.get('name')?.errors?.['maxlength']) {
+                        <div [jhiTranslate]="TRANSLATION_BASE + '.maxLengthValidationErrorName'" [translateValues]="{ max: '30' }"></div>
+                    }
+                    @if (!form.get('name')?.errors?.['required'] && !form.get('name')?.errors?.['maxlength'] && form.get('name')?.errors?.['pattern']) {
```

Comment:
Maybe I missed something, but why are you also checking for `!form.get(‘name’)?.errors?.[‘required’] && !form.get(‘name’)?.errors?.[‘maxlength’] `and not just the pattern error (`form.get('name')?.errors?.['pattern']`)?